sdk/communication/Azure.Communication.Common/tests/Identity/EntraTokenCredentialTest.cs

```diff
@@ -38,27 +41,29 @@ public void Setup()
         }
 
         [Test]
-        public void EntraTokenCredential_Init_ThrowsErrorWithNulls()
+        [TestCase(new string[] { communicationClientsScope }, typeof(ArgumentException))]
```

Comment:
It doesn't seem like we're using the exception type in any of the tests. Let's remove it from all TestCases then.